### PR TITLE
[stable/gcloud-sqlproxy] Bump image tag to 1.11

### PR DESCRIPTION
**Pull request by mandrean**
Bumps the gcloud-sqlproxy image tag and updates the README.

### CODE REVIEW

**Comment by rimusz** [comment]
please remove `stable/nginx-ingress/values.yaml` change, also there is no image tag change there nor readme change

File: stable/gcloud-sqlproxy/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: gcloud-sqlproxy
-version: 0.2.0
+version: 0.2.1
```
Comment (mattfarina) [inline stable/gcloud-sqlproxy/Chart.yaml:2]: This is a version bump to the gcloud-sqlproxy chart. Yet, there are no changes to this chart in the PR.

File: stable/nginx-ingress/values.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ controller:
   name: controller
   image:
     repository: gcr.io/google_containers/nginx-ingress-controller
-    tag: "0.9.0-beta.14"
+    tag: "0.9.0-beta.15"
```
Comment (mattfarina) [inline stable/nginx-ingress/values.yaml]: This image tag change is to something other than the title of the PR or the chart change. The corresponding changes appear to be against 2 different projects.

**Review by mattfarina — CHANGES_REQUESTED** [review]
Please fix this PR as the changes listed are for one chart but appear to be two disconnected charts without a full change in either.

**Comment by mandrean** [comment]
@rimusz @mattfarina Ah yes, thanks for the heads up. That was an accident. Should be fixed now!

**Review by mattfarina — APPROVED** [review]

**Pull request merged**